notify_listener.go

```diff
@@ -179,10 +180,21 @@ func (l *NotifyListener) processChangeset(event *store.Event) {
 		var newValues map[string]interface{}
 		err := json.Unmarshal(event.NewValues, &newValues)
 		if err != nil {
-			l.errCh <- err
+			l.errCh <- fmt.Errorf("failed to unmarshal new values: %w", err)
+		}
+
+		var newRawValues map[string]json.RawMessage
+		err = json.Unmarshal(event.NewValues, &newRawValues)
+		if err != nil {
+			l.errCh <- fmt.Errorf("failed to unmarshal raw new values: %w", err)
 		}
 
 		for k, v := range newValues {
+			switch v.(type) {
+			case map[string]interface{}:
```

Comment:
This would be a good place for the comments

Reply:
Sure I can add them.

Reply:
Done